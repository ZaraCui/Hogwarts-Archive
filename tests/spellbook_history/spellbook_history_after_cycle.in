ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Harry Potter
RENT 100000 1001
RELINQUISH 100000 1001
SPELLBOOK HISTORY 1001
EXIT

