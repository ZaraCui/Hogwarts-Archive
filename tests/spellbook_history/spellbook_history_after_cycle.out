user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: Success.

user: 100000

user: Ending Archive process.
